code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -627,6 +627,26 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/M, alien)
 /datum/reagent/medicine/bicaridine/overdose_crit_process(mob/living/M, alien)
 	M.apply_damages(1, 3, 2)
 
+/datum/reagent/medicine/meralyne
+	name = "Meralyne"
+	id = "meralyne"
+	description = "Meralyne is a concentrated form of bicardine and can be used to treat extensive blunt trauma."
+	color = "#E6666C"
+	overdose_threshold = REAGENTS_OVERDOSE
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL
+	scannable = TRUE
+
+/datum/reagent/medicine/meralyne/on_mob_life(mob/living/M, alien)
+	M.heal_limb_damage(4*REM,0)
+	..()
```

Comment:
Swap these two lines and change it to
```suggestion
	. = ..()
```

Reply:
Sadly, this causes an error where the compiler works, but the client gets massive runtime errors with this change. The old version worked upon reversion.

Reply:
What kind of runtime errors?

Reply:
I can send you part of the log I had earlier. It's in #coding in the Discord, but part of it was that it somehow caused `runtime error: Cannot read null.squads` from changing the line as above.

Reply:
Closed the connection even after a successful, "error-free" compilation.

Reply:
That makes no sense, regarding this change. Try it again.

Reply:
Alright, after clearing out the merge garbage with a fresh install, the changes seem to have worked as smooth as butter as of a preliminary test.